bitmind/base/validator.py

```diff
@@ -375,18 +378,27 @@ def save_state(self):
 
         # Save the state of the validator to file.
         np.savez(
-            self.config.neuron.full_path + "/state.npz",
+            os.path.join(self.config.neuron.full_path, "state.npz"),
```

Comment:
could this breeak anything? Why was this working previously if it needs to be changed?

Reply:
Won't break anything, tested this specifically - just for consistency with the new code I added to load/save the performance tracker